src/replication_manager.py

```diff
@@ -354,6 +361,9 @@ def _get_needed_replication_type(config, db, db_state, ha_replics):
         except Exception:
             ratio = 0.0
         if ratio >= over:
+            logging.debug("Needed repl type is async, because current load ratio %f > overload_sessions_ratio %f",
```

Comment:
`load ratio` -> `sessions ratio` to indicate exact meaning of metric (`load` term is too vague).